section2/reflection.md

```diff
@@ -1,15 +1,23 @@
+I'm now worried I missed something about the instructions because I really didn't notice a huge change here. All the directions that were left out were things that were covered previously and the directions still seemed pretty clear and direct.
 
-2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.  
+It's a statement that runs under certain conditions. If we're out of eggs go to the store and get more eggs. If I've done four pomodoros in a row my pomodoro app gives a longer break than normal.  
+5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.  
+You essentially run all the conditionals one after another and the program reads them that way. It evaluates the first condition and if it's true acts on that and if not it evaluates the next condition until it gets to an "else" statement or the conditionals stop.  
 
-3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?  
+Now you're talking logical operators. You can use && to check if multiple conditions are all true and || to check if any of several conditions are true.  
 
-4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
-
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
-
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
-
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?  
+I guess I'm wondering about nesting if statements or what happens if in more complex programs you are attaching multiple if statement trees to the same variable. (Multiple parts of gitHub might want to run checks on someone's username for instance.) Is there a way or organizing that or is it still just "whatever's first in the code evaluates first" even with multiple if statements.
```

Comment:
Yes great question. Nested conditionals do happen and are required sometimes. JavaScript reads and executes files from the top to the bottom, so that order does matter.